xapi-tracing-export.opam

```diff
@@ -0,0 +1,25 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+maintainer: ["Xapi project maintainers"]
+authors: ["xen-api@lists.xen.org"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://xapi-project.github.io/"
+bug-reports: "https://github.com/xapi-project/xen-api/issues"
+depends: [
+  "dune" {>= "2.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/xapi-project/xen-api.git"
```

Comment:
Have you tried naming the two new libraries as xapi-tracing.core and xapi-tracing.export? This allows to keep using xapi-tracing.opam

Reply:
If a new package is needed, use xapi-tracing.opam for xapi-tracing.core and create xapi-tracing-export.opam for xapi-tracing.export

Reply:
I kept the main one as `xapi-tracing` and the new one as `xapi-tracing.export`. No new package needed. Thanks!